src/common/DevHostAgent/DevHostConstants.cs

```diff
@@ -9,7 +9,7 @@ internal static class DevHostConstants
     {
         internal static class DevHostAgent
         {
-            public const int Port = 50051;
+            public const int Port = 50052;

```

Comment:
💡 idea / thoughts: This port is used with some specific use-case or is it just increment by 1 because GRPC services is using the other `50051` port? If so can just an idea that can we get something fixated for B2K cli and document it? cc: @hsubramanianaks as fyi and thoughts.

Reply:
<img width="1408" alt="Screenshot 2022-12-13 at 10 38 17 AM" src="https://user-images.githubusercontent.com/4663400/207377843-b01ee9e8-4a43-4f81-a751-44342f35e890.png">

Reply:
Sounds good, cool, lets fyi @qpetraroia  for the documentation update, or if we its something small and we want it to be part of this PR? , also, thinking out loud for idea but I hope this will not introduce any breaking change for folks who have dependency on this this for port which was used by B2K. (just in case if its day-0 port.)  

Please feel free to resolve this conversation if you are happy. thanks heaps. ☕️☕️👍